src/Symfony/Component/Workflow/Tests/MarkingStore/SubjectWithType.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Symfony\Component\Workflow\Tests\MarkingStore;
```

Comment:
missing license header (same in MethodMarkingStoreTest.php btw)

Reply:
Actually, many files (outside this component) misse the header. I'm gonna do another PR

Reply:
https://github.com/symfony/symfony/pull/46018